README.md

```diff
@@ -1,16 +1,16 @@
 # xmtp-android
 
-![Test](https://github.com/xmtp/xmtp-android/actions/workflows/test.yml/badge.svg) ![Lint](https://github.com/xmtp/xmtp-android/actions/workflows/lint.yml/badge.svg) ![Status](https://img.shields.io/badge/Project_Status-Pre--Preview-red)
+![Test](https://github.com/xmtp/xmtp-android/actions/workflows/test.yml/badge.svg) ![Lint](https://github.com/xmtp/xmtp-android/actions/workflows/lint.yml/badge.svg) ![Status](https://img.shields.io/badge/Project_Status-Developer_Preview-yellow)
 
 `xmtp-android` provides a Kotlin implementation of an XMTP message API client for use with Android apps.
 
 Use `xmtp-android` to build with XMTP to send messages between blockchain accounts, including DMs, notifications, announcements, and more.
 
-This SDK is in **Pre-Preview** status and ready for you to experiment with.
+This SDK is in **Developer Preview** status and ready for you to start building with.
 
-However, we do **not** recommend using Pre-Preview software in production apps. Software in this status is likely to change based on feedback.
+However, we do **not** recommend using Developer Preview software in production apps. Software in this status may change based on feedback.
 
-Specifically, this SDK is currently building out an example app.
+Specifically, this SDK is working to improve performance and test push notifications.
```

Comment:
Thank you for this @nplasterer  - how about something like this? =)

```suggestion
Specifically, we are working on providing push notifications and performance optimizations in the example app.
```

Reply:
```suggestion
Specifically, we are working on providing an example of push notification and performance optimizations in the example app.
```